Novo Topico inserido com o Capítulo de nome SubModulos/SubProjetos

carlosdelfino opened the pull request:
Luciano, lhe enviei um mensagem sobre a adição de um novo tópico para SubModulos e SubProjetos no Git.

Segue então o que já fiz, torço para que seja aprovado.

Grato pela atenção.
Delfino.

dukex commented:
21dcd40 - SubProjetos não precisa estar em CamelCase.
b1abd69 - Uma feature por commit por favor, seu nome deve estar no lugar correto, a lista deve estar em ordem alfabetica.

Demais commits SubProjetos em CamelCase não é necessario

carlosdelfino commented:
Estarei corrigindo, é uma mania as vezes nem percebo.

Infelizmente como este commit já foi feito conta no meu histórico, esto agora trabalhando num novo branch, e a cada finalização de etapa estarei efetuando o merge do branch de trabalho para o branch master.

Eu não sei bem como fazer o reset  branch, para remover este histórico e quando fizer o merge do novo material este não ficar com pequenos commits, você pode me ajudar?

dukex commented:
Não precisa fechar o pull request, commita as alterações que elas vão ser adicionadas ao ese pull request, e não tem problema dele ficar no historico, o git esta ai para isso, faz as alterações que acha necessarias e commit

carlosdelfino commented:
blz, vou terminar e enviar o que acabei de fazer, para meu fork,

Assim que fizer o merge te envio um novo pull request do master. ok?

carlosdelfino committed:
Correção da ordenação dos nomes do colaboradores,
Colocando o nome do Carlos Delfino no lugar correto

dukex commented:
quando commita verifica se aparece os commit nesse pull request, em cima do lado do link discussion tem um commits, se aparece seu commit lá só comentar aqui, e o pessoal aceita

carlosdelfino committed:
Ignorando arquivos de configuração do Eclipse.

carlosdelfino committed:
Merge branch 'refs/heads/master' of
git@github.com:carlosdelfino/git_na_pratica.git into HEAD

carlosdelfino committed:
Removido camelcase, adicionado conceito Subprojeto
Adicioneo o conteito Subprojeto ao nome do topico, já que o conceito
se adequa melhor do que Submodulo em se tratanto de projetos.

carlosdelfino committed:
Adicionei o termono Submodulo.
Quando fiz as pesquisas procurei pelo conceito subprojeto, e não
encontrei, porem encontrei como sendo Submodulo, mas optei manter na
listagem de referencias o Termo Subprojeto, assim fica mais dentro do
contexto prático.

carlosdelfino committed:
Outro Ajuste do camel case, e também de conceito.
Veja como os Conceitos Modulo, Submodulo e Projeto se interagem.

carlosdelfino committed:
Remoção do Camel case.

carlosdelfino commented:
os ultimos commits que fiz no master estão lá.
é so liberar.

Hoje a noite ou amanha, vou liberar outro commit, que será o capitulo já pronto, estou com ele todo escrito, agora estou fazendo a revisão, removendo textos fora do contexto e tentando melhorar a logica.

Tal material está no branch disponível neste link: https://github.com/carlosdelfino/git_na_pratica/tree/desenrolar_submodulo assim que terminar faço o commit, depois o merge para o master, e o request pull para vcs aprovarem.

dukex commented:
Nice!!

Fiz umas correções na lista de colaboradores que estava alterado e subi
